src/main/java/au/com/rayh/DeveloperProfileLoaderStep.java

```diff
@@ -0,0 +1,93 @@
+package au.com.rayh;
+
+import com.google.inject.Inject;
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepDescriptorImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution;
+import org.jenkinsci.plugins.workflow.steps.StepContextParameter;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Created by lfitzgerald on 03/11/17.
+ */
+public class DeveloperProfileLoaderStep extends AbstractStepImpl{
+
+    private String profileId;
+
+    @DataBoundConstructor
+    public DeveloperProfileLoaderStep() {}
+
+
+    @DataBoundSetter
+    public void setProfileId(String value) {
+        this.profileId = value;
+    }
+
+    private static class DeveloperProfileLoaderExecution extends AbstractSynchronousNonBlockingStepExecution<Void>{
+        private static final long serialVersionUID = 1L;
+
+        @Inject
+        private transient DeveloperProfileLoaderStep dpls;
+
+        @StepContextParameter
+        @SuppressWarnings("unused")
+        private transient Run build;
+
+        @StepContextParameter
+        @SuppressWarnings("unused")
+        private transient FilePath workspace;
+
+        @StepContextParameter
+        @SuppressWarnings("unused")
+        private transient Launcher launcher;
+
+        @StepContextParameter
+        @SuppressWarnings("unused")
+        private transient TaskListener listener;
+
+
+        @Override
+        protected Void run() throws Exception {
+
+            String profileId = dpls.profileId;
+
+            DeveloperProfileLoaderWrapper profileLoader = new DeveloperProfileLoaderWrapper(profileId);
+            profileLoader.setProjectScope(false);
+            profileLoader.perform(build, workspace, launcher, listener);
+
+            profileLoader.unload(workspace, launcher, listener);
+
```

Comment:
We can't run "unload" here otherwise the certificates will be deleted before we able to use it.

Maybe we could add a new "action" property in the DeveloperProfileLoaderStep class:

Loads the profile:
```
loadDeveloperProfile(
   profileId: 'myid',
   action: 'load'
)
```

Unloads the profile:
```
loadDeveloperProfile(
   profileId: 'myid',
   action: 'unload'
)
```

Then we would need to update the cordova Jenkinsfile to invoke the "unload" function once the build finishes running.